ui/src/app/applications/components/utils.tsx

```diff
@@ -31,6 +32,10 @@ export function nodeKey(node: NodeId) {
     return [node.group, node.kind, node.namespace, node.name].join('/');
 }
 
+export function createdOrNodeKey(node: NodeId) {
+    return node.createdAt || nodeKey(node);
```

Comment:
Can we change this to `node?.createdAt || nodeKey(node)` to prevent a crash if node is `undefined`?